extensions/src/ACRE2Mumble/MumbleCallbacks_speaking.cpp

```diff
@@ -40,6 +40,7 @@ void mumble_onUserTalkingStateChanged(mumble_connection_t connection, mumble_use
             return;
         }
 
+		// TODO: Is the TalkingState enum defined somewhere in ACRE? 'cause it is not defined in Mumble
         if (status == TalkingState::PASSIVE || status == TalkingState::INVALID) {
```

Comment:
:point_up:

Reply:
Mumble includes from that time: https://github.com/IDI-Systems/acre2/blob/a0572a4dd4fe871b8b896e544300605f4cd0d87c/extensions/src/ACRE2Mumble/mumble_includes/PluginComponents.h#L46-L53

Reply:
But that is not an enum class either so the way it is referenced should have not worked back then as well :eyes:

Reply:
Fairly sure the reason it works is because it's NOT an `enum class`, which would throw an error. With regular `enum` it works, but it is not a suggested use IIRC.